### PR TITLE
Landing page updates PROD-16

### PR DESCRIPTION
**Pull request by johnnieskywalker**

### CODE REVIEW

**Comment by johnnieskywalker** [comment]
image

**Comment by johnnieskywalker** [comment]
image

File: src/components/list-section/index.tsx [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
+    the entire lifecycle of your projects. Utilizing agile methodologies like Scrum and Kanban,
+    we provide structured planning, execution, and delivery,
+    ensuring that your products meet market demands and achieve success.`,
+  },
+
+  {
+    title: `Management Improvement`,
+    description: `Enhance your leadership capabilities and management practices
+    with our management improvement services.
+    We offer coaching and training to build effective teams, foster innovation, and drive strategic goals.
```
Comment (Denyllon) [inline src/components/list-section/index.tsx:39]: Typo: faster*

Recommendaton: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

File: src/components/list-section/index.tsx [hunk 2]
```diff
@@ -1,35 +1,43 @@
...
+     we ensure that AI technologies are effectively embedded into your business processes,
+     enhancing decision-making, efficiency, and innovation.`,
   },
   {
-    title: `OKRs AI Generator`,
-    description: `Set objectives and key results that align with your vision and drive your team forward.
-    Our OKRs Generator simplifies the process of establishing, tracking, and achieving your key business outcomes. 
-    Foster a culture of accountability and continuous improvement as you scale.`,
+    title: `Business Scaling`,
+    description: `FScale your business with confidence and precision.
```
Comment (Denyllon) [inline src/components/list-section/index.tsx]: Typo: Scale*

File: src/components/why-section/index.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  },
+];
+
+function WhySection() {
+  return (
+    <section id="features" className={tw(`bg-white pb-6`)}>
+      <div className={tw(`max-w-7xl mx-auto p-4 sm:p-6 lg:p-8`)}>
+        <div className={tw(`container mx-auto px-6 p-6 bg-white`)}>
+          <div className={tw(`mb-16 text-center`)}>
+            <p className={tw(`mt-2 text-5xl lg:text-7xl font-bold tracking-tight text-gray-900`)}>Why Chose Exegov?</p>
```
Comment (Denyllon) [inline src/components/why-section/index.tsx]: Typo: Choose*

**Review by Denyllon — CHANGES_REQUESTED** [review]

**Comment by Denyllon** [comment]
Besides the above things:
1. It seems the "Get in touch" section does not have padding over 768px
2. We could consider making the company icons smaller than right now
3. The "Why choose Exegov" section should shrink immediately into one column layout, instead of two. 

Screenshot 2024-08-07 at 14 35 30
Screenshot 2024-08-07 at 14 31 58

File: src/components/list-section/index.tsx (hunk 1 above)
Comment (johnnieskywalker) [reply src/components/list-section/index.tsx:39]: I think @bartexegov meant foster

**Pull request merged**